src/suricata.c

```diff
@@ -1493,6 +1494,7 @@ static TmEcode ParseCommandLine(int argc, char** argv, SCInstance *suri)
         {"pcap-buffer-size", required_argument, 0, 0},
         {"unittest-filter", required_argument, 0, 'U'},
         {"list-app-layer-protos", 0, &list_app_layer_protocols, 1},
+        {"list-protocols", 0, &list_protocols, 1},
```

Comment:
I worry that ppl will misinterpret this to mean the protocols Suricata supports in general. Should we make it something like '--list-detect-protocols' or something?

Reply:
I didn't find a better name, also thought about that. But if you think that one is less confusing I will change it :) PR will come